### PR TITLE
[DPE-4861] Add test to ensure correct k8s endpoints created for clusters with the same name

### PR DESCRIPTION
**Pull request by shayancanonical**
## Issue
We are missing an integration test for https://github.com/canonical/mysql-k8s-operator/pull/501

## Solution
Add integration test

### CODE REVIEW

**Review by paulomach — APPROVED** [review]
:heart_eyes:

File: tests/integration/high_availability/test_k8s_endpoints.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+MYSQL_CLUSTER_ONE = "mysql1"
+MYSQL_CLUSTER_TWO = "mysql2"
+MYSQL_CLUSTER_NAME = "test_cluster"
+TEST_APP_ONE = "mysql-test-app1"
+TEST_APP_TWO = "mysql-test-app2"
+
+
+@pytest.mark.group(1)
+@pytest.mark.abort_on_fail
+async def test_labeling_of_k8s_endpoints(ops_test: OpsTest):
```
Comment (paulomach) [inline tests/integration/high_availability/test_k8s_endpoints.py:29]: the one thing to dispute here is the test being under ha tests, but I guess it's due usage of helpers, and not about being ha. we may need to shuffle some of the helpers for organization sake

**Review by taurus-forever — APPROVED** [review]
Thank you!

File: tests/integration/high_availability/test_k8s_endpoints.py (hunk 1 above)
Comment (shayancanonical) [reply tests/integration/high_availability/test_k8s_endpoints.py:29]: agreed, let's rearrange furniture in a separate PR allowing a better classification of integration tests

**Pull request merged**